.github/workflows/tests.yml

```diff
@@ -12,8 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7]  #  TODO: Enable additional versions: 3.7
-        gcc-version: [8]  #  TODO: Enable additional versions: 6, 7
```

Comment:
Could you please remind me why gcc-related installations are removed? Are these not being used anymore in favor of gxx?

Reply:
Yeah, now the compilers are installed with conda so this isn't necessary. I actually changed that a while ago, deleting these lines is just cleanup.